base/test/audioToTextXform_tests.cpp

```diff
@@ -0,0 +1,59 @@
+#include "stdafx.h"
+#include <boost/test/unit_test.hpp>
+#include "FrameMetadata.h"
+#include "FrameMetadataFactory.h"
+#include "Frame.h"
+#include "Logger.h"
+#include "test_utils.h"
+#include "PipeLine.h"
+#include "FileWriterModule.h"
+#include "FileReaderModule.h"
+#include "FileWriterModule.h"
+#include "AudioToTextXForm.h"
+#include "Module.h"
+#include<iostream>
+#include<fstream>
+#include<vector>
+
+BOOST_AUTO_TEST_SUITE(audioToTextXform_test)
```

Comment:
I have a general question here: so, is it possible to quantify how much audio is processed in each `step()` call of the module ?

Reply:
Depends on bufferSize set in props. If you send a chunk>bufferSize it will process all of it.